frontend/src/Pages/MyProgress.tsx

```diff
@@ -205,29 +205,31 @@ export default function MyProgress() {
                                         </tr>
                                     </thead>
                                     <tbody className="flex flex-col gap-4 mt-4">
-                                        {certData?.map((certificate: any) => {
-                                            return (
-                                                <tr
-                                                    className="flex flex-row justify-between body-small items-center"
-                                                    key={Math.random()}
-                                                >
-                                                    <td className="w-1/2">
-                                                        {
-                                                            certificate.program_name
-                                                        }
-                                                    </td>
-                                                    <td className="w-1/5 flex">
-                                                        {new Date(
-                                                            certificate.created_at.split(
-                                                                'T'
-                                                            )[0]
-                                                        ).toLocaleDateString(
-                                                            'en-US'
-                                                        )}
-                                                    </td>
-                                                </tr>
-                                            );
-                                        })}
+                                        {certData?.map(
+                                            (certificate: Outcome) => {
+                                                return (
+                                                    <tr
+                                                        className="flex flex-row justify-between body-small items-center"
+                                                        key={Math.random()}
```

Comment:
Can do that in most cases, but this enum that is used in MyCourses does not have an id... 
![image](https://github.com/user-attachments/assets/0006581c-1b8a-4983-9e57-b4e37db92487)

Reply:
It would be the Outcome (certificate) that would need the ID in this particular instance.
 
but if there is one where you cannot use an ID, then use the index of the array you are calling `map` on